src/Diagrams/TwoD/Transform.hs

```diff
@@ -242,15 +250,24 @@ reflectXY = transform reflectionXY
 --   the point @p@ and direction @d@.
 reflectionAbout :: OrderedField n => P2 n -> Direction V2 n -> T2 n
 reflectionAbout p d =
-  conjugate (rotationTo d <> translation (origin .-. p))
+  conjugate (rotationTo (reflectY d) <> translation (origin .-. p))
             reflectionY
 
+
```

Comment:
Is this `combine` function used anywhere?  I think it was part of code deleted in your last commit.